starcounter-include.html

```diff
@@ -411,7 +416,7 @@
                     // Just the composition was changed
                     // and it's different than already stored one
                     subPath === this.compositionProvider + '.Composition' &&
-                    value != this.storedLayout
+                    value != this._storedLayout
```

Comment:
Maybe you'd like to rename 

- `_storedLayout` -> `_storedComposition`
- `_forceLayoutChange` -> `_forceRenderComposition`

Reply:
I'd be happy to, but I'm not sure   id `_forceLayoutChange` is not used by some other elements.

Reply:
It is only used by us https://github.com/search?q=_forceLayoutChange&type=Code. But I agree, let's keep `_forceLayoutChange` for now.